src/Appwrite/SMS/Adapter/Dialog360.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Appwrite\SMS\Adapter;
+
+use Appwrite\SMS\Adapter;
+
+// Reference Material
+// https://docs.360dialog.com/docs/whatsapp-api/whatsapp-api/sandbox#php
+
+class Dialog360 extends Adapter
+{
+    /**
+     * @var string
+     */
+    private string $endpoint = 'https://waba.360dialog.io/v1/messages';
+
+    /**
+     * @param string $from
+     * @param string $to
+     * @param string $message
+     * @return void
+     */
+    public function send(string $from, string $to, string $message): void
+    {
+        $to = ltrim($to, '+');
+
+        $this->request(
+            method: 'POST',
+            url: $this->endpoint,
+            headers: ["Content-Type: application/json",
+                      "D360-Api-Key: {$this->secret}"],
+            payload: json_encode([
+                'recipient_type' => 'individual',
+                'to' => $to,
+                'type' => 'template',
+                'template' => [
+                    'namespace' => $from,
+                    'name' => "{$this->user}",
+                    'language' => [
+                        'policy' => 'deterministic',
+                        'code' => 'en'
```

Comment:
In the messaging library, language can be another parameter to the constructor of the adapter.

Reply:
Yes, I will do it. Thanks for the suggestion.